src/pyqasm/visitor.py

```diff
@@ -2186,6 +2244,10 @@ def visit_statement(self, statement: qasm3_ast.Statement) -> list[qasm3_ast.Stat
             qasm3_ast.SubroutineDefinition: self._visit_subroutine_definition,
             qasm3_ast.ExpressionStatement: lambda x: self._visit_function_call(x.expression),
             qasm3_ast.IODeclaration: lambda x: [],
+            qasm3_ast.WhileLoop: self._visit_while_loop,
+            # Add break/continue if present in AST
+            getattr(qasm3_ast, "BreakStatement", None): self._visit_break_statement,
+            getattr(qasm3_ast, "ContinueStatement", None): self._visit_continue_statement,
```

Comment:
I think we can remove these lines, as I mentioned in an [earlier comment](https://github.com/qBraid/pyqasm/pull/199#discussion_r2115518683)